src/main/java/com/strandls/document/ApiConstants.java

```diff
@@ -43,5 +43,8 @@
 	public static final String UPLOAD = "/upload";
 	public static final String USERGROUP = "/usergroup";
 	public static final String V1 = "/v1";
+	public static final String GNRD="/gnfinder";
+	public static final String NAMES="/names";
+	public static final String DELETENAME="/deleteName";
```

Comment:
path should be small case , instead of camel case

Reply:
Corrected.